plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -595,8 +596,9 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
                 SchemaParser.toJson(icebergTable.schema()),
                 PartitionSpecParser.toJson(icebergTable.spec()),
                 getColumns(icebergTable.schema(), typeManager),
-                getDataPath(icebergTable.location()),
-                getFileFormat(icebergTable));
+                icebergTable.location(),
```

Comment:
change from `getDataPath(icebergTable.location())` to `icebergTable.location()` doesn't seem to belong to "pass in table properties to initialize location provider in sink prov…" commit